nnpdf_data/nnpdf_data/commondata/CMS_Z0J_8TEV/filter.py

```diff
@@ -0,0 +1,40 @@
+'''
+Filter script for CMS_Z0J_8TEV
+'''
+
+import logging
+
+from filter_utils import Extractor
+import numpy as np
+import yaml
+
+logging.basicConfig(level=logging.INFO, format='[%(levelname)s] %(message)s')
+
+
+def check_dat_with_legacy(observable, rtol=1e-03):
+    """
+    Same as `check_unc_with_legacy`, but for central data points.
```

Comment:
`check_unc_with_legacy` lives in a different directory at the same level so I don't think it makes much sense to refer to it here

Reply:
I kept it because I thought it could have been helpful, but it is unnecessary here. I'll delete it.